docs/userguide/miscellaneous.rst

```diff
@@ -1,3 +1,5 @@
+.. _Automatic Resource Extraction:
```

Comment:
I had to re-add some labels that have been removed in the original refactoring to make such places linkable. We need to consider enabling sphinx's built-in autolabel extension instead. But this would be a separate effort because it would involve cleaning everything up and making sure there are no title duplicates.

cc @alvyjudy